site/src/atomics/typography.md

```diff
@@ -26,6 +27,7 @@ The typography scale is designed for great readability across the platform. This
 | `letter-spacing`  | `wide`                                                           | N\A        |
 | `text-transform`  | `uppercase`                                                      | N\A        |
 | `text-align`      | `center`, `right`                                                | `tablet`   |
+| `line-height`     | `1`, `normal`                                                    | N\A        |
```

Comment:
curious what the use case for `line-height-1` is. seems like we could do without it